dotcom-rendering/src/client/ga/index.ts

```diff
@@ -1,44 +1,62 @@
 import {
 	getConsentFor,
+	onConsent,
 	onConsentChange,
 } from '@guardian/consent-management-platform';
 import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
 import { loadScript, log } from '@guardian/libs';
 import { init, sendPageView } from './ga';
 
-export const ga = (): Promise<void> => {
+/** Memoize loading to prevent loading twice */
+let loadedGoogleAnalytics = false;
```

Comment:
I really like the pattern of prefixing boolean var names with `is`:
```suggestion
let isLoaded = false;
```
i find it helps when you're skim-reading unfamiliar code